### ISSUE TITLE
A workflow data dump?

### ISSUE THREAD

**Issue by mschwamb** [zooniverse/Panoptes#1045]
I thought this was more of a backend issue. Apologies if this is a duplicate issue, I tried to search for it in the open tickets on the front end and here, and didn't find a match. I'm looking at the csv file of subjects classified, and the it gives the workflow id and classification responses in the annotations column look lik e '{"task":"init","value":[1,2],"task_label":"What features do you see in the image? (**Select all that apply**)"}]'' 

I'd have to know what the text was for options 1 and 2 especially if later on we changed the interface. We could track that because of the workflow id associated with the classification, but there's currently no option to download a history of the workflow structure so i can tell what responses 1 and 2 were for  workflow 10.3 

I think the ability to download the workflow history would be useful for science teams doing their own analysis. I'm not sure how feasible that is though a csv file or in another format

**Comment by marten** [comment]
It's probably related to #979. We made a quick change to add labels for some tasks, but missed the multiple-choice case. We also figured the correct fix would be basically what you're saying too: a complete reference from identifiers to labels (instead of adding the labels to the classifications themselves). That would be in JSON format (same as the raw classification data). That takes a little more time to implement, so we held off on that until after the launch (which has now passed).

**Comment by camallen** [comment]
@mschwamb once we get the latest code deployed the classification dump annotations should respect the workflow version and include the task question and answer labels. Can you confirm that the outputs are working as expected in respect to workflow versions? 

Also will this suffice or do we still need a workflow version manifest dump as well?

**Comment by mschwamb** [comment]
@camallen I still think there should be a separate workflow version manifest if that info is easy to dump out into some form. Personally, I like having a separate file and not having to go through the annotations for that information. If I made a change or added workflow knowing when I did that would be useful and if there are 1.5 million classifications or more having to unique the workflow parts of the classification dump can be slow to sort and unique.  I've found having a separate subjects file has made things just easier so I think having a json/csv dump of the workflow history would be useful.  

Also I think it's useful for someone writing their own reduction software since they can look and quickly see what the allowed responses were now and previously in one quick glance if there is a workflow version manifest dump
